.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+  release:
+    types: [ published, created, edited ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - uses: jakejarvis/s3-sync-action@master
+        name: Deploy non-tag branches
+        if: github.event_name == 'pull_request'
+        with:
+          args: --acl public-read --follow-symlinks --delete --cache-control "max-age=60"
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: 'eu-west-1'
+          SOURCE_DIR: 'dist'
+          DEST_DIR: ${{ github.head_ref }}
+
+      - uses: jakejarvis/s3-sync-action@master
+        name: Deploy released branches
+        if: github.event_name == 'release'
+        with:
+          args: --acl public-read --follow-symlinks --delete --cache-control "max-age=2592000"
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: 'eu-west-1'
+          SOURCE_DIR: 'dist'
+          DEST_DIR: ${{ github.ref }}
+
+      - uses: jakejarvis/s3-sync-action@master
+        name: Deploy latest release
+        if: github.event_name == 'push'
```

Comment:
This block should also be the release condition, it just uploads to the `release` folder instead of a `<branchname>` folder.

Reply:
Mmmh, ok I finally figured this out.
So this another deploy always overwrites `/release` with the latest release.
Will do.